wallet/service.go

```diff
@@ -142,11 +142,8 @@ func (service *Service) LinkWallet(
 			return handlers.WrapError(errors.New("wallets do not match"), "unable to match wallets", http.StatusForbidden)
 		}
 	} else {
-		destination := info.ProviderID
-		if info.ProviderID == "" {
-			destination = tx.Destination
-		}
-		err := service.Datastore.LinkWallet(info.ID, destination, providerLinkingID, anonymousAddress, depositProvider)
+		// tx.Destination will be stored as "ProviderID"
```

Comment:
Is this comment still true? It looks like we're inserting into the db as `user_deposit_destination` but there's not an associated migration. A case of bad find/replace?

Reply:
will correct.  thanks.

Reply:
https://github.com/brave-intl/bat-go/pull/469/commits/32ffbbfdd0dc32ff53aa77414a0d9d7cd24b276c